Toolset/palettes/script editor/behaviors/revsestackbehavior.livecodescript

```diff
@@ -41,6 +41,12 @@ local sLastShownExecutionPoint
 #   Sets the current object to edit to pObject. If pStartingLine is given, then seeks to that line
 #   otherwise seeks to the last selected line.
 command revSESetCurrentObject pObject, pStartingLine, pDontSendCallbacks
+   lock screen
+   
+   revSEGetCurrentObject
+   if the result is not empty and the long id of pObject is the long id of the result then
```

Comment:
@livecodeali we really need an equality operator for object references... it's a bit frustrating to need to do stuff like this to ensure you are comparing apples with apples.